python/ray/_private/serialization.py

```diff
@@ -435,6 +435,15 @@ def _deserialize_object(self, data, metadata, object_ref):
                 else:
                     actor_id = None
                 return ActorUnavailableError(error_info.error_message, actor_id)
+            elif error_type == ErrorType.Value(
```

Comment:
The receiver task execution handler reads the data from `in_actor_object_store` which is NCCL recv by `fetch_p2p_dependency_handler`.